tests/utils/test_common.py

```diff
@@ -20,6 +20,14 @@ def test_raise_warning():
     assert record[0].message.args[0] == "My warning."
 
```

Comment:
It would be good to rename the previous test `test_raise_warning()` to indicate that the `UserWarning` is tested, e.g. `test_raise_user_warning()`